### DISCUSSION TITLE
compile problem

### DISCUSSION THREAD

**Discussion by kaifeng0306** [ceres-solver/ceres-solver#1006] [Q&A]
When I was compiling the ceres-solver-1.14.x, I met a problem with compiling. Wish for your help, thanks a lot.
The problem is :
/usr/bin/ld: /usr/local/lib/libopenblas.a(dormlq.f.o): undefined reference to symbol '_gfortran_concat_string@@GFORTRAN_7'
/usr/bin/ld: /usr/lib/x86_64-linux-gnu/libgfortran.so.4: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
examples/CMakeFiles/curve_fitting.dir/build.make:122: recipe for target 'bin/curve_fitting' failed
make[2]: *** [bin/curve_fitting] Error 1
CMakeFiles/Makefile2:4230: recipe for target 'examples/CMakeFiles/curve_fitting.dir/all' failed

**Comment by sandwichmaker** [comment]
This is not a ceres issue. This looks like a problem with your fortran compiler/linker config.
